AWSAppSyncClient/AWSAppSyncClient.swift

```diff
@@ -126,7 +126,7 @@ public class AWSAppSyncClient {
                 let prefixTrackerValue = appSyncConfig.url.absoluteString + "_" + authTypeString
                 if let (clientString, clientCount) = AWSAppSyncClient.prefixTracker[prefixTrackerKey] {
                     if clientString != prefixTrackerValue {
-                        throw AWSAppSyncClientConfigurationError.invalidClientDatabasePrefixConfiguration("Configured two clients with the same database prefix")
+                        throw AWSAppSyncClientConfigurationError.clientDatabasePrefixAlreadyInUse("Configured two clients with the same database prefix")
```

Comment:
recommend: `AWSAppSyncClientConfigurationError.cacheConfiguraitonAlreadyInUse`
This will reduce the need to add another enum for future validations where 2 clients have no prefix